eclair-core/src/main/scala/fr/acinq/eclair/io/PeerConnection.scala

```diff
@@ -357,6 +364,9 @@ class PeerConnection(keyPair: KeyPair, conf: PeerConnection.Conf, switchboard: A
       case Event(ResumeAnnouncements, d: ConnectedData) =>
         log.info(s"resuming processing of network announcements for peer")
         stay() using d.copy(behavior = d.behavior.copy(fundingTxAlreadySpentCount = 0, ignoreNetworkAnnouncement = false))
+
+      case Event(KillIdle, d: ConnectedData) if !d.isPersistent =>
+        stop(FSM.Normal)
```

Comment:
nit: it would be cleaner to handle the message in all cases, to avoid a warning log if the connection became persistent (because a channel was opened):

```suggestion
      case Event(KillIdle, d: ConnectedData) =>
          if (!d.isPersistent) {
              stop(FSM.Normal)
          } else {
              stay()
          }
```

Reply:
Done